pyaerocom/data/variables.ini

```diff
@@ -1173,7 +1173,7 @@ var_name = emibvoc
 description = Biogenic NMVOC emissions
 standard_name = tendency_of_atmosphere_mass_content_of_biogenic_nmvoc_expressed_as_carbon_due_to_emission
 var_type = Emission fluxes
-unit = kg m-2 s-1
+unit = mg C m-2 h-1
```

Comment:
Should we apply the same rule here with var_name? i.e. name it emiCbvoc?

Reply:
I set the unit to be the same as in the netcdf files to avoid unit conversion. This is not ideal, but I want to leave it like this. I'll think about the naming convention